src/common/chip/Chip.tsx

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
 import classNames from 'classnames';
+import { StatusLabel, StatusLabelProps } from 'hds-react';
 
 import styles from './chip.module.scss';
 
 export interface ChipProps {
-  color: 'blue' | 'red' | 'green' | 'yellow' | 'orange' | 'grey';
+  type: StatusLabelProps['type'] | 'warning';
   label: string;
   className?: string;
 }
 
-const Chip = ({ color, label, className }: ChipProps) => {
-  return <span className={classNames(styles.chip, styles[color], className)}>{label}</span>;
+const Chip = ({ type, label, className }: ChipProps) => {
```

Comment:
These are a little different than they were, but that's the evolution of HDS. Looks good. Should we rename this to `StatusLabel`?

Reply:
Renamed to `StatusLabel`